specification/app/resource-manager/Microsoft.App/preview/2022-01-01-preview/ContainerApps.json

```diff
@@ -309,6 +309,62 @@
           }
         }
       }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.App/containerApps/{containerAppName}/analyzeCustomHostName": {
+      "post": {
+        "tags": [
+          "ContainerApps"
+        ],
+        "summary": "Analyzes a custom hostname for a Container App",
+        "operationId": "ContainerAppsRevisions_AnalyzeCustomHostName",
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "name": "containerAppName",
+            "in": "path",
+            "description": "Name of the Container App.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "customHostname",
+            "in": "query",
+            "description": "Custom hostname.",
+            "type": "string"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/CustomHostnameAnalysisResult"
+            }
+          },
+          "404": {
+            "description": "Not found.",
+            "x-ms-error-response": true
+          },
```

Comment:
Under what conditions would this be a 404 ? Since it is a tracked resource the parent resource {containerappname} resource not existing would cause a failure at ARM level itself

Reply:
I don't think it should ever return 404 @vinisoto ?

Reply:
Correct. Removed in the latest commit